src/terminal/InputGenerator.h

```diff
@@ -496,4 +496,67 @@ struct formatter<terminal::MouseTransport>
         return format_to(_ctx.out(), "<{}>", unsigned(_value));
     }
 };
+
+template <>
+struct formatter<terminal::Key>
+{
+    template <typename ParseContext>
+    constexpr auto parse(ParseContext& ctx)
+    {
+        return ctx.begin();
+    }
+    template <typename FormatContext>
+    auto format(terminal::Key _value, FormatContext& _ctx)
+    {
+      std::vector<std::string> key_values = {"F1","F2",
+	"F3",
```

Comment:
You can try running clang-format over this file to fix the indentation here. 

The command on the CLI would be something like: `clang-format -i  ./src/terminal/InputGenerator.h`